### ISSUE TITLE
Support optionally a) separated repos for pages and media and b) additional repos for core php runtime and for conf

### ISSUE THREAD

**Issue by mhoffrog** [woolfg/dokuwiki-plugin-gitbacked#55]
### Feature request:
I would like to configure:
1. separate repos for pages and for media
2. additional repo for the php core runtime path
3. additional repo for the conf path

### Motivation for:
1. It is a great benefit of DW that it manages the pages in plain text format. So I would like to avoid to pollute the pages repo with binary artifacts => it would be good to maintain a separate git repo for the media files.
2. The core runtime for a DW instance typically requires some lifecycle management in terms of plugin installations, plugin upgrades, core upgrades etc.. So I would like to keep track of those changes by gitbacked as well.
3. As the DW configuration is a critical part of the availability of a DW instance, I would like to let gitbacked keep track of any configuration change as well.

### Solution:
As I did implement this kind of extension I will create an appropriate pull request soon:

![gitbacked_separate_repos_01](https://user-images.githubusercontent.com/45703410/113296262-858c2b80-92f9-11eb-99a6-e15f94591d5e.png)

**Comment by woolfg** [comment]
thanks @mhoffrog for the PR, looks good. So it should be backward compatible, right? If people update and do not have values for the options, everything should still work?

**Comment by mhoffrog** [comment]
Yes @woolfg, this is correct. None of the "Optional:" marked configuration properties must be set. If those are left empty, then anything works as before. So it is backward compatible with a probable existing configuration from an installed former version of this plugin.

**Comment by woolfg** [comment]
Hey @mhoffrog, as my time is very limited and I am currently not actively using DokuWiki and the plugin, would you be interested in becoming a maintainer and helping with the plugin? Especially testing of pull requests (mostly yours ;) ) is very time consuming when you don't use the the wiki yourself. Feel free to contact me (see options on gassler.dev ) - cheers, Wolfgang
